README.md

```diff
@@ -31,3 +31,25 @@ trademarks or logos is subject to and must follow
 [Microsoft's Trademark & Brand Guidelines](https://www.microsoft.com/en-us/legal/intellectualproperty/trademarks/usage/general).
 Use of Microsoft trademarks or logos in modified versions of this project must not cause confusion or imply Microsoft sponsorship.
 Any use of third-party trademarks or logos are subject to those third-party's policies.
+
+## Testing
+
+ginko is used for unit tests.
+
+### setting up goproxy
+ado token: https://dev.azure.com/msazure/_usersSettings/tokens
+```
+vim ~./zshrc # bashrc 
+export AKS_GOPROXY_TOKEN=******
+export GOPROXY=https://<username>:$AKS_GOPROXY_TOKEN@goproxyprod.goms.io
+soruce ~./zshrc
+```
+
+
+Unit tests are available inside client/pkgs/
+### To run the full suite
+```
+tls_bootstrapping/aks-secure-tls-bootstrap/client/pkg % go test ./... -coverprofile cover.out 
```

Comment:
we have make targets for this, e.g. `make test` / `make test-coverage`

Reply:
lol I don't know the target names apparently:

https://github.com/Azure/aks-secure-tls-bootstrap/blob/ce0c5a995107c192369e5e6012c5288e30ae36d0/Makefile#L53C1-L65C180